rewrite-java/src/main/java/org/openrewrite/java/ChangePackage.java

```diff
@@ -91,6 +100,19 @@ public TreeVisitor<?, ExecutionContext> getVisitor() {
                 stopAfterPreVisit();
                 if (tree instanceof JavaSourceFile) {
                     JavaSourceFile cu = (JavaSourceFile) tree;
+                    if (Boolean.TRUE.equals(visitStringLiterals)) {
+                        AtomicBoolean found = new AtomicBoolean(false);
+                        Traits.literal().asVisitor(lit -> {
+                            String string = lit.getString();
+                            if (string != null && string.contains(oldPackageName)) {
+                                found.set(true);
+                            }
+                            return lit.getTree();
+                        }).visit(cu, ctx, getCursor().getParentTreeCursor());
+                        if (found.get()) {
+                            return SearchResult.found(cu);
+                        }
```

Comment:
Personally I tend to prefer the reduce function where possible; in this case you're only interested in the boolean, and reduce makes that explicit
```suggestion
                        if (Traits.literal().asVisitor((Literal lit, AtomicBoolean bool) -> {
                                    String string = lit.getString();
                                    if (string != null && string.contains(oldPackageName)) {
                                        bool.set(true);
                                    }
                                    return lit.getTree();
                                }).reduce(cu, new AtomicBoolean(false), getCursor().getParentTreeCursor())
                                .get()) {
                            return SearchResult.found(cu);
                        }
```